### ISSUE TITLE
Request for the model file.

### ISSUE THREAD

**Issue by bangboomy** [KPeng9510/OS-SAR#2]
Can you provide trained model files?

**Comment by KPeng9510** [comment]
Hi, we can share the logits file which is in the google drive https://drive.google.com/file/d/1izv8KxjUgUuzBFqm9h6QQ7hSoPeZbCKL/view?usp=sharing. Thank you!

**Comment by KPeng9510** [comment]
Btw, this logits file is for CTRGCN NTU60 run 1 for CV evaluation.
